scripts/test/generate_lld_tests.py

```diff
@@ -27,58 +27,54 @@ def files_with_extensions(path, extensions):
       yield file, ext
 
 
-def generate_object_files(clang_bin):
-  print '\n[ building object files from C sources... ]\n'
+def generate_wast_files(clang_bin, lld_bin, emscripten_root):
+  print '\n[ building wasm files from C sources... ]\n'
```

Comment:
is it wasm or wast?  the print statement and the function name seem to disagree.